tests/unit/test_mcp_gateway.py

```
@@ -281,3 +281,718 @@ def test_empty_allowed_tools_fails(self, tmp_path):
+        line = captured.err.strip()
+        import json
+
+        rec = json.loads(line)
+        assert rec["ev"] == "handshake"
+        assert rec["level"] == "INFO"
+        assert rec["agent"] == "a"
+        assert rec["intent"] == "i"
+        assert rec["decision"] == "allow"
+        assert rec["sid"] == "s1"
+        assert "ts" in rec
+        # タイムスタンプの精度（マイクロ秒を含む ISO 8601 形式: YYYY-MM-DDTHH:MM:SS.mmmmmmZ）
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)